src/lib/features/client-feature-toggles/delta/client-feature-toggle-delta.ts

```diff
@@ -197,6 +200,9 @@ export class ClientFeatureToggleDelta {
         }
     }
 
+    /**
```

Comment:
We needed a way to clear out delta between e2e test runs. We might find a better way in future.